fastparquet/writer.py

```diff
@@ -902,10 +890,15 @@ def write(filename, data, row_group_offsets=50000000,
         before writing, potentially providing a large speed
         boost. The length applies to the binary representation *after*
         conversion for utf8, json or bson.
-    append: bool (False)
-        If False, construct data-set from scratch; if True, add new row-group(s)
-        to existing data-set. In the latter case, the data-set must exist,
-        and the schema must match the input data.
+    append: bool (False) or 'overwrite'
+        - If False, construct data-set from scratch; if True, add new row-group(s)
+          to existing data-set. In the latter case, the data-set must exist,
+          and the schema must match the input data.
+        - If 'overwrite', existing data can be replaced. To enable this, these other
```

Comment:
"If 'overwrite', existing partitions will be replaced in-place, where the given data has any rows within a given partition"

Reply:
done